python/lsst/ts/linearstage/mocks/mock_zaber_lst.py

```diff
@@ -19,14 +19,315 @@
+        except asyncio.IncompleteReadError as e:
+            # Used for debugging received message
+            self.log.info(f"{e.partial}")
+            raise
+        self.log.info(f"{command=} received.")
+        reply: str = self.device.parse_message(command)
+        self.log.debug(f"Writing {reply=}.")
+        await self.write_str(reply)
+
+
+class MockLSTV2:
+    """Mock the zaber stage.
+
+    Attributes
+    ----------
+    address : `int`
+        The device address.
+    id : `int`
+        The ID number of the device.
+    system_serial : `int`
+        The serial number of the system.
+    version : `str`
+        The version string of the firmware.
+    version_build : `int`
+        The version build number of the firmware.
+    modified : `bool`
+        Has the device been modified?
+    axis_count : `int`
+        The number of axes attached to the device.
+    message_id : `int` | `None`
+        The ID of the current message.
+    max_packet : `int`
+        The number of bytes allowed in a packet.
+    max_word : `int`
+        The max size of a word.
+    position : `lsst.ts.simactuators.PointToPointActuator`
+        The encoder position along the stage.
+    axes : `types.SimpleNamespace`
+        The information from each axis.
+    """
+
+    def __init__(self) -> None:
+        self.address: int = 1
+        self.id: int = 30342
+        self.system_serial: int = 115313
+        self.version: str = "7.32"
+        self.version_build: int = 13718
+        self.modified: bool = False
+        self.axis_count: int = 1
+        self.message_id: int | None = None
+        self.max_packet: int = 80
+        self.max_word: int = 50
+        self.position: simactuators.PointToPointActuator = (
+            simactuators.PointToPointActuator(
+                min_position=0, max_position=15000000, speed=60000
+            )
+        )
+        self.axes: types.SimpleNamespace = types.SimpleNamespace(
+            axis1=types.SimpleNamespace(
+                address=1,
+                id=68404,
+                resolution=64,
+                modified=False,
+            ),
+            axis2=types.SimpleNamespace(
+                address=2, id=0, resolution="NA", modified=False
+            ),
+            axis3=types.SimpleNamespace(
+                address=3, id=0, resolution="NA", modified=False
+            ),
+            axis4=types.SimpleNamespace(
+                address=4, id=0, resolution="NA", modified=False
+            ),
+        )
+
+    def parse_message(self, msg) -> str:
+        """Parse the message.
+
+        Parameters
+        ----------
+        msg : `str`
+            The mesage received from the client.
+
+        Returns
+        -------
+        `str`
+            The reply that was generated.
+        """
+        msg = msg.lstrip("/")
+        msg = msg.split(" ")
+        count = len(msg)
+        request = types.SimpleNamespace()
+        if count == 3:
```

Comment:
Maybe use a match statement instead? Also, please extract the code for each case to a dedicated method to improve readability of the code.

Reply:
Thanks for using a match statement here. I still think that extracting the code for each case into a dedicated method will improve readability so please do that.